src/main/java/com/sonymobile/tools/gerrit/gerritevents/workers/Coordinator.java

```diff
@@ -32,12 +32,6 @@
  * @author Robert Sandell &lt;robert.sandell@sonyericsson.com&gt;
  */
 public interface Coordinator {
-    /**
-     * Retrieves the work queue for workers to poll.
-     * @return the queue
-     */
-    BlockingQueue<Work> getWorkQueue();
```

Comment:
No, unfortunately we can't change binary signature like this. Something else needs to be done.